### PR TITLE
Update uber image part in README of both English and Chinese versions

### PR DESCRIPTION
**Pull request by olivershen-wow**
## Pull Request Checklist

- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Code compiles correctly with all tests are passed.
- [X] I've read the [contributing guide](https://github.com/microsoft/HydraLab/blob/main/CONTRIBUTING.md#making-changes-to-the-code) and followed the recommended practices.
- [X] [Wikis](https://github.com/microsoft/HydraLab/wiki) or [README](https://github.com/microsoft/HydraLab/blob/main/README.md) have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?
*If this introduces a breaking change for Hydra Lab users, please describe the impact and migration path.*

- [ ] Yes
- [X] No

## Pull Request Description

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Build related changes
- [ ] Refactoring (no functional changes, no api changes)
- [X] Code style update (formatting, renaming) or Documentation content changes
- [ ] Other (please describe): 

**A few words to explain your changes**:

Update README of both English and Chinese versions for uber image part.

### How you tested it
*Please make sure the change is tested, you can test it by adding UTs, do local test and share the screenshots, etc.*

### Feature UI screenshots or Technical design diagrams
*If this is a relatively large or complex change, kick it off by drawing the tech design with PlantUML and explaining why you chose the solution you did and what alternatives you considered, etc...*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,10 +55,6 @@ Please visit our **[GitHub Project Wiki](https://github.com/microsoft/HydraLab/w
 <span id="quick-start"></span>
 ### Quick guide on out-of-box Uber docker image
 
-Hydra Lab uses [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/) as cloud file storage solution to persist log files, video, app package, etc. Please go to your Azure portal, open an Azure blob storage account, get the [connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string),
-and place it in the environment variable with the name of BLOB_CONNECTION_STR. Brief steps: [Login Azure](https://azure.microsoft.com/) -> [Portal](https://portal.azure.com/#home) -> [Storage Accounts](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Storage%2FStorageAccounts) -> Create new storage account (you may disable the public access for the container) -> In the created storage account, find `Access Keys` tab -> copy `Connection string`. 
-![image](https://user-images.githubusercontent.com/8344245/216729523-387dc162-54d8-41dd-b136-f2e3c780b10a.png)
```
Comment (hydraxman) [inline README.md:60]: 这部分不要删掉，对于采用azure blob的团队 还是有用的，可以找一wiki存放

Comment (hydraxman) [reply README.md:60]: 这里留一句，如果你采用azure blob作为存储方案，可以参考这篇文档：link

File: README.md [hunk 2]
```diff
@@ -67,13 +63,30 @@ Hydra Lab offers an out-of-box experience of docker image called Uber. By provid
...
+Hydra Lab use local file system as default storage, no extra environment variable is needed:
+
+```bash
+docker run -p 9886:9886 --name=hydra-lab ghcr.io/microsoft/hydra-lab-uber:latest
+```
+
+**2. use third-party storage service**
+
+Hydra Lab currently supports [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/) as cloud file storage solution to persist log files, video, app package, etc. Please go to your Azure portal, open an Azure blob storage account, get the [connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string),
+and place it in the environment variable with the name of BLOB_CONNECTION_STR. Brief steps: [Login Azure](https://azure.microsoft.com/) -> [Portal](https://portal.azure.com/#home) -> [Storage Accounts](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Storage%2FStorageAccounts) -> Create new storage account (you may disable the public access for the container) -> In the created storage account, find `Access Keys` tab -> copy `Connection string`.
```
Comment (hydraxman) [inline README.md:81]: 可以采用chatgpt优化文本表达

Comment (olivershen-wow) [reply README.md:81]: 这一块就是上面的内容移下来的，没有改动内容

File: README.zh-CN.md [hunk 3]
```diff
@@ -62,15 +59,32 @@ Hydra Lab 提供了一个名为 Uber 开箱即用的 docker 镜像。在简单
 docker pull ghcr.io/microsoft/hydra-lab-uber:latest
 ```
 
-**第2步. 在你的机器上运行，并使用 BLOB_CONNECTION_STR 作为参数**
+**第2步. 在你的机器上运行**
+
+直接从以下两种命令中选择一种，以开始你在 Hydra Lab 的体验：
```
Comment (AC-Phoenix) [inline README.zh-CN.md:64]:
```suggestion
只需选择以下其中一个命令，即可开始您在Hydra实验室的体验：
```

File: README.zh-CN.md [hunk 4]
```diff
@@ -62,15 +59,32 @@ Hydra Lab 提供了一个名为 Uber 开箱即用的 docker 镜像。在简单
...
+Hydra Lab 默认使用本地文件系统作为存储，不需要额外的环境变量：
+
+```bash
+docker run -p 9886:9886 --name=hydra-lab ghcr.io/microsoft/hydra-lab-uber:latest
+```
+
+**2. 使用第三方存储服务**
 
-可以通过提供包含 `BLOB_CONNECTION_STR=${YOUR_BLOB_CONNECTION_STR}` 的配置文件env.properties在运行目录，然后通过如下方式传入文件相对路径参数：
+Hydra Lab 现在支持 [Azure Blob 存储](https://azure.microsoft.com/en-us/products/storage/blobs/) 作为云文件存储解决方案，以持久化存储日志文件、视频、应用包等。请访问你的 Azure 门户，创建一个 Azure Blob 存储账户，获取 [connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string)，
```
Comment (AC-Phoenix) [inline README.zh-CN.md:76]:
```suggestion
Hydra Lab 目前支持 [Azure Blob 存储](https://azure.microsoft.com/en-us/products/storage/blobs/) 作为云文件存储解决方案，以持久化存储日志文件、视频、应用程序包等。请访问您的 Azure 门户，创建一个 Azure Blob 存储账户，获取 [connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string)，
```

File: README.zh-CN.md [hunk 5]
```diff
@@ -62,15 +59,32 @@ Hydra Lab 提供了一个名为 Uber 开箱即用的 docker 镜像。在简单
...
+并将其放入环境变量中，名称为 BLOB_CONNECTION_STR。简要步骤：[Login Azure](https://azure.microsoft.com/) -> [Portal](https://portal.azure.com/#home) -> [Storage Accounts](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Storage%2FStorageAccounts) -> 创建新的存储账户（你可以限制容器的公共读写） ->在创建的存储账户中, 找到 `Access Keys` 页面 -> 复制 `Connection string`。
```
Comment (AC-Phoenix) [inline README.zh-CN.md:77]:
```suggestion
并将其存放在名为 BLOB_CONNECTION_STR 的环境变量中。简要步骤：[登录 Azure](https://azure.microsoft.com/) -> [Portal](https://portal.azure.com/#home) -> [Storage Accounts](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Storage%2FStorageAccounts) -> 创建新的存储账户（你可以限制容器的公共读写） ->在所创建的存储账户中, 找到 `Access Keys` 页面 -> 复制 `Connection string`即可。
```

File: README.md [hunk 6]
```diff
@@ -67,13 +63,30 @@ Hydra Lab offers an out-of-box experience of docker image called Uber. By provid
...
+
+Hydra Lab use local file system as default storage, no extra environment variable is needed:
+
+```bash
+docker run -p 9886:9886 --name=hydra-lab ghcr.io/microsoft/hydra-lab-uber:latest
+```
+
+**2. use third-party storage service**
+
+Hydra Lab currently supports [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/) as cloud file storage solution to persist log files, video, app package, etc. Please go to your Azure portal, open an Azure blob storage account, get the [connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string),
```
Comment (AC-Phoenix) [inline README.md:80]:
```suggestion
Hydra Lab currently supports [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/) as cloud file storage solution to persist various data types such as log files, video, app package, etc. Please go to your Azure portal, open an Azure blob storage account, get the [connection string](https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string),
```

**Review by AC-Phoenix — APPROVED** [review]

**Review by AC-Phoenix — APPROVED** [review]

**Pull request merged**